api/src/backend/api/v1/views.py

```diff
@@ -1039,6 +1076,32 @@ def get_queryset(self):
 
         return queryset
 
+    @action(detail=False, methods=["get"], url_name="findings_services_regions")
+    def findings_services_regions(self, request):
+        try:
+            updated_at = datetime.strptime(
+                request.query_params["filter[updated_at]"], "%Y-%m-%d"
+            ).date()
+            if updated_at > date.today():
+                raise ValidationError(detail="The date must be a date in the past.")
+        except ValueError:
+            raise ValidationError(detail="Invalid date format.")
```

Comment:
Was this asked by UI or something? We have a lot of date filters without this kind of logic. It simply should return no values at all if we provide a future date. If it is required, this validation should belong in the filter module in my opinion.